### PR TITLE
Update etcd plugin documentation wrt etcdv3 support

### PR DESCRIPTION
**Pull request by nitisht**
### 2. Which issues (if any) are related?
See #1960

### CODE REVIEW

File: plugin/etcd/README.md [hunk 1]
```diff
@@ -107,6 +113,7 @@ etcd skydns.local {
 ...
 ~~~
 
+Before getting started with examples, please setup `etcdctl` (with etcdv3 API) as explained [here](https://coreos.com/etcd/docs/latest/dev-guide/interacting_v3.html). This will help you put sample keys to you etcd server easily.
```
Comment (miekg) [inline plugin/etcd/README.md]: Mention that `curl <something>` doesn't work anymore with etcd3?

**Comment by miekg** [comment]
thanks for tackling this, btw!

**Comment by nitisht** [comment]
> thanks for tackling this, btw!

No problem :)

File: plugin/etcd/README.md (hunk 1 above)
Comment (nitisht) [reply plugin/etcd/README.md]: I updated the paragraph, PTAL

File: plugin/etcd/README.md [hunk 2]
```diff
@@ -107,6 +113,9 @@ etcd skydns.local {
 ...
 ~~~
 
+Before getting started with examples, please setup `etcdctl` (with `etcdv3` API) as explained [here](https://coreos.com/etcd/docs/latest/dev-guide/interacting_v3.html). This will help you put sample keys to you etcd server easily.
```
Comment (miekg) [inline plugin/etcd/README.md]: with *these*

Comment (miekg) [reply plugin/etcd/README.md]: "to you etcd" -> you to put sample keys in your etcd server.

File: plugin/etcd/README.md [hunk 3]
```diff
@@ -107,6 +113,9 @@ etcd skydns.local {
...
+
+If you prefer, you can use `curl` to populate `etcd` server, but with `curl` the endpoint gateway depends on the version of `etcd`. For instance, `etcd v3.2` or before uses only [CLIENT-URL]/v3alpha/* while `etcd v3.5` or later uses [CLIENT-URL]/v3/* . Also, Key and Value must be base64 encoded in JSON. With, `etcdctl` these details are automatically taken care.
```
Comment (miekg) [inline plugin/etcd/README.md]: populate *the*

gateway -> URL

Comment (miekg) [reply plugin/etcd/README.md]: in *the* JSON *payload*

Comment (miekg) [reply plugin/etcd/README.md]: taken care *off*

File: plugin/etcd/README.md (hunk 2 above)
Comment (nitisht) [reply plugin/etcd/README.md]: done

File: plugin/etcd/README.md (hunk 3 above)
Comment (nitisht) [reply plugin/etcd/README.md]: done

**Review by miekg — APPROVED** [review]

**Pull request merged**